process_pr.py

```diff
@@ -1022,13 +1029,29 @@ def process_pr(
     repository = repo.full_name
     repo_org, repo_name = repository.split("/", 1)
     auto_test_repo = AUTO_TEST_REPOS
+
+    global CMSSW_CATEGORIES
+    CMSSW_CATEGORIES = copy.deepcopy(default_CMSSW_CATEGORIES)
+
+    # LEGACY_CATEGORIES is a mapping from name to datetime (when the category becomes legacy)
+    # Extract categories that are legacy at the moment of issue creation
+    lc = getattr(repo_config, "LEGACY_CATEGORIES", {})
+    issue_created_at: datetime = issue.created_at
+    if issue_created_at.tzinfo is None:
+        issue_created_at = issue_created_at.replace(tzinfo=timezone.utc)
+
+    for lc in (cat for cat, ts in lc.items() if issue_created_at > ts):
```

Comment:
@iarspider , may be change either `lc` loop variable name  here or rename `lc` variable on line 1038.

Reply:
Done